src/RapidezStatamicServiceProvider.php

```diff
@@ -55,6 +55,31 @@ public function boot()
         Alternates::register();
     }
 
+    public function bootSites() : self
+    {
+        $sites = [];
+        $stores = Rapidez::getStores();
+
+        foreach ($stores as $store) {
+            $sites[$store['code']] = [
+                'name' => $store['name'] ?? $store['code'],
+                'locale' => '{{ config:rapidez.statamic.sites.' . $store['code'] . '.locale }}',
+                'lang' => '{{ config:rapidez.statamic.sites.' . $store['code'] . '.lang }}',
+                'url' => '{{ config:rapidez.statamic.sites.' . $store['code'] . '.url }}',
```

Comment:
We can also get this from Magento right? The locales and url's are also configured there?